javascript/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "javascript",
+  "version": "1.0.0",
+  "description": "",
+  "main": "src/index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
We can remove "test" and replace by "migrate", "sensor" and "dev" to run the commands like so: npm run <command> [-- <args>]
example:  "migrate": "node src/cmd/migrate/index.js"
Then update README.md instructions: npm run migrate --hosts $NODE1